src/neon/generator.rs

```diff
@@ -1,7 +1,8 @@
 use crate::value::generator::ESCAPED;
 use std::io;
-use crate::neon::intrinsics::*;
 use crate::neon::stage1::neon_movemask;
+use std::arch::aarch64::*;
```

Comment:
this leaves a sour taste...

Reply:
I kind of feel we don't want to re-implement it all, that'll make it really hard to use simd-lite with std and that feels just wrong.

Reply:
rock on, we can save that for another time